src/rpc/rawtransaction_util.cpp

```diff
@@ -221,6 +221,9 @@ UniValue SignTransaction(CMutableTransaction& mtx, const UniValue& prevTxsUnival
                     // Automatically also add the P2WSH wrapped version of the script (to deal with P2SH-P2WSH).
                     keystore->AddCScript(GetScriptForWitness(witnessScript));
                 }
+                if (rs.isNull() && ws.isNull()) {
+                    throw JSONRPCError(RPC_INVALID_PARAMETER, "Missing redeemScript/witnessScript");
+                }
```

Comment:
I feel like this has already been merged

Reply:
Yeah, I just rebased it (` ajtowns force-pushed the ajtowns:201906-signrawerror branch from 073a2f8 to f7e4676`) but github's not picking up the new HEAD?

Reply:
It is well known for GitHub to show outdated branches in the web view (see also https://github.com/bitcoin/bitcoin/issues/15847#issuecomment-484871831)


I guess you have to force push twice?

Reply:
Looks like it's doing the right thing now (I pushed a dummy commit and force-pushed it out of existence). You should be seeing f7e4676591c9027a6fe4dd1ebc9e1ed35b27574f as HEAD.